enchant.lic

```diff
@@ -0,0 +1,176 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#enchant
+=end
+
+custom_require.call(%w[common common-arcana events common-crafting events])
+
+class Enchant
+  include DRC
+  include DRCC
+  include DRCA
+
+  def initialize
+    @settings = get_settings
+    @book_type = "artificing"
+    @bag = @settings.crafting_container
+    @bag_items = @settings.crafting_items_in_container
+    @belt = @settings.enchanting_belt
```

Comment:
enchanting_belt: should be added to profiles/base.yaml, not there are any :)

Reply:
Done!